src/Microsoft.DotNet.Files/ContentFiles.cs

```diff
@@ -19,45 +20,68 @@ public ContentFiles(ProjectContext context)
             _context = context;
         }
 
-        public void StructuredCopyTo(string targetDirectory)
+        public void StructuredCopyTo(string targetDirectory, IEnumerable<IncludeEntry> includeEntries)
         {
-            var sourceFiles = _context
-                .ProjectFile
-                .Files
-                .GetContentFiles();
+            if (includeEntries == null)
```

Comment:
I agree but it feels like an unnecessary change given the old code is deprecated and will be removed anyway.